src/font/mod.rs

```diff
@@ -73,13 +79,25 @@ impl Font {
     }
 
     pub fn data(&self) -> &[u8] {
-        (*self.data).as_ref()
+        #[cfg(not(feature = "peniko"))]
+        {
+            (*self.data).as_ref()
+        }
+        #[cfg(feature = "peniko")]
+        {
+            self.data.data.data()
+        }
     }
 
     pub fn rustybuzz(&self) -> &RustybuzzFace<'_> {
         self.rustybuzz.borrow_dependent()
     }
 
+    #[cfg(feature = "peniko")]
+    pub fn as_peniko(&self) -> PenikoFont {
+        self.data.clone()
```

Comment:
This clones an underlying `Arc` in [Blob](https://github.com/linebender/peniko/blob/615caa3fdb0df9e8396e4efe4ab89fe8e1a872a7/src/blob.rs#L13)